src/test/java/salesforce/cases/CreateCasesTests.java

```diff
@@ -0,0 +1,28 @@
+package salesforce.cases;
+
+import org.testng.annotations.Test;
+import salesforce.base.BaseTest;
+import salesforce.config.EnvironmentConfig;
+import salesforce.ui.PageTransporter;
+import salesforce.ui.pages.CasesFormPage;
+import salesforce.ui.pages.CasesPage;
+import salesforce.ui.pages.HomePage;
+import salesforce.ui.pages.LoginPage;
+
+public class CreateCasesTests extends BaseTest {
+    private PageTransporter pageTransporter = new PageTransporter();
+    private CasesPage casesPage;
+    private HomePage homePage;
+    private CasesFormPage casesFormPage;
+
+    @Test
```

Comment:
The test dos not asserts nothing

Reply:
asserts are added on the next PR. On the page abstract task.